test/mage/core_test.clj

```diff
@@ -13,37 +14,29 @@
                      [" -h"] [" --help"]
                      ["  -h"] ["  --help"]]
           :let [cmd (str "./bin/mage " (str/join " " help-cmds))]]
-    (println (str "Testing that bin/mage has help with '" (pr-str cmd) "'"))
-    (let [out (u/sh cmd)]
-      (when-not (str/includes? out "The following tasks are available:")
-        (System/exit 1)))))
+    (testing (format "'%s'" (pr-str cmd))
+      (let [out (u/sh cmd)]
+        (when-not (str/includes? out "The following tasks are available:")
+          (System/exit 1))))))
 
 (defn- bb-task-has-example? [task-name]
   (doseq [cmd [(str "./bin/mage " task-name " -h")
                (str "./bin/mage " task-name " --help")]]
-    (println (str "Testing that task has examples with '" cmd "'"))
-    (when-not (str/includes? (u/sh cmd) "Examples:")
-      (System/exit 1))))
+    (testing (format "'%s'" cmd)
```

Comment:
Don't you want to provide more context in this testing string?

Reply:
Oh nevermind, see where you added it.

Reply:
well it kind of provides it, with `testing` call in `deftest`?..

Reply:
exactly - i just missed that at first because it was further down in the PR